code/client/scripts/index.js

```diff
@@ -1207,24 +1207,23 @@ function ShareLink(channel, entity, product_imgurl, product_name, product_brand)
 
                 if (!(productId || tags || minPrice || maxPrice)){
                     //Sharing base webpage
-                    product_imgurl = window.location.origin + $("img.header-logo").attr("src");
-                    pin_desc = "Find Your Perfect Bag, We Make It Easy.";
+                    product_imgurl = encodeURI(window.location.origin + $("img.header-logo").attr("src"));
+                    pin_desc = escape("Find Your Perfect Bag, We Make It Easy.");
```

Comment:
This should use `encodeURIComponent()` instead of `escape`.  A good description of which to use and when can be found here: http://stackoverflow.com/a/3608791/879046